lib/catalog_of_my_things/genre.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class Genre
+  attr_reader :items
+  attr_accessor :id, :name
+
+  def initialize(name, id = Random.rand(1..1_000_000))
+    @name = name
+    @id = id
+    @items = []
+  end
+
+  def to_json(*args)
+    {
+      json.create_id => self.class.name,
+      'name' => name,
+      'id' => id,
+      'items' => @items
+    }.to_json(*args)
+  end
```

Comment:
👍 Great method! I think this will help us a lot in storing the elements as JSON.